db-postgres : Upgrade v3.0.0-beta.67 -> v3.0.0-beta.69 throws error

### Link to reproduction

_No response_

### Payload Version

3.0.0-beta.69

### Node Version

20 LTS

### Next.js Version

15 canary 58

### Describe the Bug

After upgrading from beta 67 to beta 69 it's not possible anymore for me locally to run any CLI commands.

here the list of my payload packages:

```
"@payloadcms/db-postgres": "3.0.0-beta.69",
"@payloadcms/email-nodemailer": "3.0.0-beta.69",
"@payloadcms/graphql": "3.0.0-beta.69",
"@payloadcms/next": "3.0.0-beta.69",
"@payloadcms/plugin-form-builder": "3.0.0-beta.69",
"@payloadcms/plugin-nested-docs": "3.0.0-beta.69",
"@payloadcms/plugin-redirects": "3.0.0-beta.69",
"@payloadcms/plugin-search": "3.0.0-beta.69",
"@payloadcms/plugin-seo": "3.0.0-beta.69",
"@payloadcms/richtext-lexical": "3.0.0-beta.69",
"next": "15.0.0-canary.58",
"nodemailer": "^6.9.14",
"payload": "3.0.0-beta.69",
"sharp": "^0.33.4",
"tsx": "^4.7.1"
```

Seems like a dependency is missing or some package export is not correct?

`npm run payload migrate:create` yields:

```
node:internal/process/promises:391
    triggerUncaughtException(err, true /* fromPromise */);
    ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/app/cms/node_modules/@payloadcms/drizzle/src/createTableName.js' imported from /app/cms/node_modules/@payloadcms/db-postgres/dist/init.js
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at nextResolve (node:internal/modules/esm/hooks:866:28)
    at resolve (file:///app/cms/node_modules/payload/dist/bin/loader/index.js:86:12)
    at async nextResolve (node:internal/modules/esm/hooks:866:22)
    at async Hooks.resolve (node:internal/modules/esm/hooks:304:24)
    at async handleMessage (node:internal/modules/esm/worker:196:18) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///app/cms/node_modules/@payloadcms/drizzle/src/createTableName.js'
}

Node.js v20.16.0
```

I use npm as a package manager.

The issue is also not resolved if I add `"@payloadcms/drizzle": "3.0.0-beta.69",` to the dependency list.

### Reproduction Steps

See above.

### Adapters and Plugins

db-postgres

--- a/packages/db-postgres/src/init.ts
+++ b/packages/db-postgres/src/init.ts
@@ -1,14 +1,12 @@
-/* eslint-disable no-param-reassign */
-import type { SanitizedCollectionConfig } from 'payload'
-import type { Init } from 'payload'
+import type { Init, SanitizedCollectionConfig } from 'payload'
 
+import { createTableName } from '@payloadcms/drizzle'
 import { pgEnum, pgSchema, pgTable } from 'drizzle-orm/pg-core'
 import { buildVersionCollectionFields, buildVersionGlobalFields } from 'payload'
 import toSnakeCase from 'to-snake-case'
 
 import type { PostgresAdapter } from './types.js'
 
-import { createTableName } from '../../drizzle/src/createTableName.js'
 import { buildTable } from './schema/build.js'
 
 export const init: Init = function init(this: PostgresAdapter) {

